### ISSUE TITLE
Support Lua on Windows phone

### ISSUE THREAD

**Issue by PeqNP** [cocos2d/cocos2d-x#13761]
I would love to see Lua on Windows phone. We are in this process of porting our games, which are all written in Lua, to the platform and realized that Lua on the Windows phone platform is unsupported. 

I wouldn't mind doing the work to complete this. I wanted to know 1) who I can talk to help with this work? 2) does anyone know platform specific issues that may give me trouble when trying to port the Lua bindings? 3) has someone already started the work to complete this feature?

Also, I've heard talk about Windows Universal apps where the build process builds for all platforms. Is this work done? If it is, does that mean that Lua will work on the phone Without special build steps?

Any help would be appreciated. 

Thank you!

**Comment by stammen** [comment]
Hi Eric,

The cocos2d community briefly added lua support for Windows Phone 8.0 in cocos2d-x v 2.x but that work did not continue into cocos2d-x v3.x.

If you are interested in doing the port I would be happy to guide you. Please feel free to contact me directly by email.

Thanks,

Dale

**Comment by PeqNP** [comment]
Yes. I am very interested in doing this. I'll shoot you an email as soon as I am able. Thank you, Dale. 

Eric

**Comment by stammen** [comment]
Great! Talk with you soon.

**Comment by dannyjiajia** [comment]
how is the progress about this plan?

**Comment by markovitch57** [comment]
Really waiting for this. Any progress? Can help if required.
